web/sites/default/settings.php

```diff
@@ -2,264 +2,80 @@
-    'password' => getenv('MARIADB_PASSWORD') ?: 'drupal',
-    'host' => getenv('MARIADB_HOST') ?: 'mariadb',
-    'port' => 3306,
-    'charset' => 'utf8mb4',
-    'collation' => 'utf8mb4_general_ci',
-  ];
-}
-
-// Lagoon Solr connection
-if (getenv('LAGOON')) {
-  $config['search_api.server']['backend_config']['connector_config']['host'] = getenv('SOLR_HOST') ?: 'solr';
-  $config['search_api.server']['backend_config']['connector_config']['path'] = '/solr/' . getenv('SOLR_CORE') ?: 'drupal';
-}
-
-// Lagoon Varnish & reverse proxy settings
-if (getenv('LAGOON')) {
-  $varnish_control_port = getenv('VARNISH_CONTROL_PORT') ?: '6082';
-  $varnish_hosts = explode(',', getenv('VARNISH_HOSTS') ?: 'varnish');
-  array_walk($varnish_hosts, function (&$value, $key) use ($varnish_control_port) {
-    $value .= ":$varnish_control_port";
-  });
-
-  $settings['reverse_proxy'] = TRUE;
-  $settings['reverse_proxy_addresses'] = array_merge(explode(',', getenv('VARNISH_HOSTS')), ['varnish']);
-  $settings['varnish_control_terminal'] = implode($varnish_hosts, " ");
-  $settings['varnish_control_key'] = getenv('VARNISH_SECRET') ?: 'lagoon_default_secret';
-  $settings['varnish_version'] = 4;
-}
-
-// Redis configuration.
-if (getenv('LAGOON') && (getenv('ENABLE_REDIS'))) {
-  $settings['redis.connection']['interface'] = 'PhpRedis';
-  $settings['redis.connection']['host'] = getenv('REDIS_HOST') ?: 'redis';
-  $settings['redis.connection']['port'] = 6379;
-
-  $settings['cache_prefix']['default'] = getenv('LAGOON_PROJECT') . '_' . getenv('LAGOON_GIT_SAFE_BRANCH');
+// Set by Drupal core.
+/* @var $app_root string */
 
-  # Do not set the cache during installations of Drupal
-  if (!drupal_installation_attempted()) {
-    $settings['cache']['default'] = 'cache.backend.redis';
+// Path to standardized platform-wide settings includes.
+/* @var $govcms_includes string */
 
-    // Include the default example.services.yml from the module, which will
-    // replace all supported backend services (that currently includes the cache tags
-    // checksum service and the lock backends, check the file for the current list)
-    $settings['container_yamls'][] = 'modules/contrib/redis/example.services.yml';
+// Set to 'local', 'development', 'production'.
+/* @var $govcms_env string */
 
-    // Allow the services to work before the Redis module itself is enabled.
-    $settings['container_yamls'][] = 'modules/contrib/redis/redis.services.yml';
+$govcms_includes = $app_root . '/../vendor/govcms/scaffold-tooling/drupal/settings';
 
-    // Manually add the classloader path, this is required for the container cache bin definition below
-    // and allows to use it without the redis module being enabled.
-    $class_loader->addPsr4('Drupal\\redis\\', 'modules/contrib/redis/src');
+// Todo, this logic to be reviewed by Amazee.
+$govcms_is_prod = !getenv('DEV_MODE') && (getenv('LAGOON_ENVIRONMENT_TYPE') && getenv('LAGOON_ENVIRONMENT_TYPE') == 'production');
```

Comment:
No need for additional `()`
```
$govcms_is_prod = !getenv('DEV_MODE') && getenv('LAGOON_ENVIRONMENT_TYPE') && getenv('LAGOON_ENVIRONMENT_TYPE') == 'production';
```

Reply:
While i restructured the settings files, a lot of these specific lines I intentionally didn't touch. There will be some regressions to sort out too.